### DISCUSSION TITLE
About the IfcCostItem

### DISCUSSION THREAD

**Discussion by bimintelligence** [GeometryGym/GeometryGymIFC#86] [Q&A]
Hi everyone!
I‘m a little confused about the initialization of IfcCostItem.
And can it to be directed to assigned to process using IfcRelAssignsToControl?

#region Assembly GeometryGymIFC, Version=0.1.22.0, Culture=neutral, PublicKeyToken=null
#endregion

namespace GeometryGym.Ifc
{
    public class IfcCostItem : IfcControl
    {
        public IfcCostItemTypeEnum PredefinedType { get; set; }

        protected override string BuildStringSTEP(ReleaseVersion release);
    }
}

**Comment by jmirtsch** [comment]
Hi,

Thanks for reporting this.  It seems that the constructors presently have an internal access modifier, so when
referencing the dll (such as nuget) this can't be used.  I've just refactored the constructors to make them public, this will be in 
the next build.  If you need it urgently, please let me know.

Thanks,

Jon
